phpmd.xml

```diff
@@ -10,7 +10,9 @@
         <exclude name="ElseExpression" />
         <exclude name="StaticAccess" />
     </rule>
-    <rule ref="rulesets/codesize.xml" />
+    <rule ref="rulesets/codesize.xml">
+        <exclude name="TooManyPublicMethods" />
```

Comment:
Apparently, there is too much public methods on `ProductManager`. This simply exclude the check, but maybe there is a better solution for this problem.